js/main.js

```diff
@@ -31,4 +33,41 @@ Site = {
   },
 };
 
+Site.News = {
+  init: function() {
+    var _this = this;
+
+    _this.shuffleContainer = document.getElementById('shuffle-container');
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    imagesLoaded(_this.shuffleContainer, function() {
+      _this.initShuffle();
+    });
+  },
+
+  initShuffle: function() {
+    var _this = this;
+
+    $('#shuffle-preloader').remove();
+    $(_this.shuffleContainer).css('opacity', 1)
```

Comment:
ok. but isnt it better practice to add/remove classes than change fixed props with jquery?